adafruit_bitmap_font/pcf.py

```diff
@@ -133,27 +139,27 @@ def __init__(self, f, bitmap_class):
         )
 
     @property
-    def ascent(self):
+    def ascent(self) -> int:
         """The number of pixels above the baseline of a typical ascender"""
         return self._ascent
 
     @property
-    def descent(self):
+    def descent(self) -> int:
         """The number of pixels below the baseline of a typical descender"""
         return self._descent
 
-    def get_bounding_box(self):
+    def get_bounding_box(self) -> tuple:
```

Comment:
Blinka currently supports Python 3.7, so we can't use the builtin generics (introduced in 3.8).  Instead, you'll have to import `Tuple` from `typing` and use that.

Additionally, since it's a generic, you can also annotated the tuple's contents, in this case:

```suggestion
    def get_bounding_box(self) -> Tuple[int, int, int, int]:
```